convert/convert/converters/base_format/models/annotation.py

```diff
@@ -28,7 +28,7 @@ class Data(BaseModel):
 class Obj(BaseModel):
     id: int
     type: str
-    bbox: List[float]
+    bbox: Tuple[float, float, float, float]
```

Comment:
It's okay for now, but later I think we should define simple type alias for this